meta-mbl-distro/recipes-core/packagegroups/packagegroup-mbl-development.bb

```diff
@@ -25,6 +25,7 @@ PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " python3-pygobject"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " gobject-introspection"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " e2fsprogs"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " mbed-crypto-test"
+PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " psa-storage"
```

Comment:
I don't think we should put `psa-storage` in our package lists - it's a library that shouldn't be added to the image unless something depends on it.

If we have a separate package for the sample app then I think putting the example app in the dev package group would be okay, but the library should be pulled in implicitly as a dependency.

Reply:
>> I don't think we should put psa-storage in our package lists - it's a library that shouldn't be added to the image unless something depends on it.

mbed-crypto-test will depend on it.

Also what depends on mbed-crypto-test (nothing at present) which has been added?

I need this to be present in the PR for the time being for testing psa_trusted_storage_linux PRs.

Reply:
> Also what depends on mbed-crypto-test (nothing at present) which has been added?

Nothing, but mbed-crypto-test is a collection of binaries rather than libraries.

> I need this to be present in the PR for the time being for testing psa_trusted_storage_linux PRs.

Won't those PRs add packages that RDEPEND on psa-trusted-storage-linux? If they do then the library will be added to the image implicitly. If they don't, then I don't see how having psa-trusted-storage-linux in the image helps.

Reply:
I'm in accordance with @jonathanhaigh-arm.

Reply:
Changes in https://github.com/ARMmbed/meta-mbl/pull/619.

Reply:
The changes in #619 leave the package in the packagegroup.